File: dayFour/python/target.py
array1= [8,-2,4]

def array (array1, number):

    sum=0

    nArr=  []
    for value in array1:
        for value2 in array1:
 
            sum= value + value2
            if sum==number:
                nArr.append(value)
                
                break;

               
            
         

    return nArr;


print(array(array1,6));

File: dayFour/javaScript/target.js
array1= [8,-2,4]

function  array (array1, number){

let sum=0;

 nArr=  [];

    for(let index=0; index<array.length; index++){

        for(let index2=0; index2<array.length; index2++){
                
                sum= array1[index]+ array1[index2];
                if(sum==number){
            
                    nArr[1]=(array1[index]);
                    nArr[0]=(array1[index2]);
                    break;

         }

          }
     

    }


return nArr;



}

module.exports = { array };
